wolfssl/ssl.h

```diff
@@ -1524,6 +1528,10 @@ WOLFSSL_API void wolfSSL_ASN1_STRING_free(WOLFSSL_ASN1_STRING* asn1);
 WOLFSSL_API int wolfSSL_ASN1_STRING_set(WOLFSSL_ASN1_STRING* asn1,
                                                   const void* data, int dataSz);
 WOLFSSL_API unsigned char* wolfSSL_ASN1_STRING_data(WOLFSSL_ASN1_STRING*);
+#ifdef WOLFSSL_QT
+WOLFSSL_API const unsigned char* wolfSSL_ASN1_STRING_get0_data(
```

Comment:
Should this be general to OPENSSL_EXTRA and not just WOLFSSL_QT?

Reply:
Yes, that is true. Should be general to OPNESSL_EXTRA